src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php

```diff
@@ -84,6 +92,23 @@ protected function triggerEvents(EventResourceRepositoryPluginInterface $plugin,
         $this->triggerEventsAll($plugin);
     }
 
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourceBulkRepositoryPluginInterface $plugin
+     * @param int[] $ids
+     *
+     * @return void
+     */
+    protected function triggerEventsBulk(EventResourceBulkRepositoryPluginInterface $plugin, array $ids = []): void
```

Comment:
This function does magic depending on parameters.
Also, it will still trigger too many events if I pass too many IDs.

Reply:
This is the expected behaviour to trigger the events as much as you want for ids. for example if you want to trigger events for ProductAbstract and id 1,2,3,4...